app/pods/components/agenda/agenda-detail/sidebar-item/component.js

```diff
@@ -1,136 +1,61 @@
-import Component from '@ember/component';
-import { computed, observer } from '@ember/object';
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
 import { inject as service } from '@ember/service';
 import { alias } from '@ember/object/computed';
+import { tracked } from '@glimmer/tracking';
 
-export default Component.extend( {
-  store: service(),
-  sessionService: service(),
-  currentSessionService: service('current-session'),
-  agendaService: service(),
-  toaster: service(),
-  classNameBindings: [
-    'isActive:vlc-agenda-detail-sidebar__sub-item--active',
-    'isClickable::not-clickable',
-    'agendaitem.retracted:vlc-u-opacity-lighter',
-    'isPostponed:vlc-u-opacity-lighter',
-    'isNew:vlc-agenda-items__sub-item--added-item'
-  ],
-  tagName: 'a',
-  selectedAgendaItem: alias('sessionService.selectedAgendaItem'),
-  currentAgenda: alias('sessionService.currentAgenda'),
-  isClickable: true,
-  hideLabel: true,
-  isShowingChanges: null,
-
-  init() {
-    this._super(...arguments);
-    observer(
-      'agendaitem.postponedTo',
-      async function () {
-        const postponed = await this.get('agendaitem.postponedTo');
-        if (!this.get('isDestroyed')) {
-          this.set('isPostponed', !!postponed);
-        }
-      }
-    );
-  },
-
-  formallyOk: computed('agendaitem.formallyOk', function () {
-    return this.agendaitem.get('formallyOk');
-  }),
-
-  agenda: computed('agendaitem', function () {
-    return this.get('agendaitem.agenda.name');
-  }),
+export default class SidebarItem extends Component {
+  /**
+   * INFO arguments from parent.
+   * @agendaitem={{agendaitem}}
+   * @selectAgendaItem={{action "selectAgendaItemAction"}}
+   */
 
-  documents: computed('agendaitem.documentVersions.@each', function () {
-    if (this.get('selectedAgendaItem')) {
-      return;
+  @service store;
+  @service sessionService;
+  @service('current-session') currentSessionService;
+  @service agendaService;
+  @service toaster;
+
+  @alias('sessionService.selectedAgendaItem') selectedAgendaItem;
+  @alias('sessionService.currentAgenda') currentAgenda;
+  @alias('args.agendaitem.checkAdded') isNew;
+
+  hideLabel = true;
+  isShowingChanges = null;
+
+  @tracked isClickable = true;
+  @tracked isPostponed = this.args.agendaitem.postponedTo.get('postponed');
```

Comment:
Ik begrijp het nog niet helemaal. Kan jij hier eens mee naar kijken @rafael-mindreau  ? Het lijkt me in ieder geval dat dit functioneel niet hetzelfde doet als wat er stond. (`!!postponed` vs `postponed.get('postponed')`). Misschien helpt het als je de redenering hierachter wat toelicht @brechtPhilips ?